README.md

```diff
@@ -7,7 +7,7 @@
 ## Announcements
 
 * This project is being maintained by [Fabian Sauter](https://github.com/com8) and [Kilian Traub](https://github.com/KingKili).
-* For quick help, and discussion libcpr also offer a [gitter](https://gitter.im/libcpr/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link) chat.
+* For quick help, and discussion libcpr also offers a [gitter](https://gitter.im/libcpr/community?utm_source=share-link&utm_medium=link&utm_campaign=share-link) chat.
```

Comment:
This stuff should not be included. As the other PR is now merged, try rebasing on the latest master or drop the commit if possible.

Reply:
Was a mistake on my side, I didn't want to include it here...